.github/workflows/ci.yml

```
@@ -1,21 +1,81 @@
 
 permissions:
   contents: read
 
 jobs:
-  build:
-    name: Lint and Build
+  # Native App Checks (Release 1.0 Focus)
+  native-checks:
+    name: Native App (Typecheck & Lint)
     runs-on: ubuntu-latest
+on:
```

actionlint (1.7.10)

key "on" is duplicated in "workflow" section. previously defined at line:3,col:1

(syntax-check)